src/main/webapp/groups.js

```diff
@@ -10,3 +10,14 @@ function init() {
 function toggleHidden() {
     goog.dom.classlist.toggle(newGroupContainer, "hidden");
 }
+
```

Comment:
Sorry I didn't see this before, but there shouldn't be any double quotation marks in JS. Prefer to use single quotes. Can you fix it throughout teh file? ie "/groups" should be '/groups'.